app/views/course/users/students.json.jbuilder

```diff
@@ -15,4 +15,10 @@ end
 
 json.manageCourseUsersData do
   json.partial! 'tabs_data', current_course: current_course
+
+  json.timelines do
+    current_course.reference_timelines.each do |timeline|
```

Comment:
If timeline designer is disabled, should we only show the default one and we dont need to let users to change?

Reply:
Actually, I think we can just exclude `timelines` altogether if the Multiple Reference Timelines component is disabled.

Reply:
Yeah thats fine